client/src/pages/ChefProfile.js

```diff
@@ -184,35 +194,68 @@ function ChefProfile(props) {
+                                >
+                                    {chefInfo.primaryAddress &&
+                                        `${chefInfo.primaryAddress.city}, ${chefInfo.primaryAddress.country}`}
+                                </Typography>
+                            </Grid>
+                            <Grid
+                                item
+                                container
+                                spacing={2}
+                                direction="row"
+                                justify="center"
+                                alignItems="center"
+                            >
+                                {chefInfo.chefProfile.cuisineSpecialty.length > 0
+                                    ? chefInfo.chefProfile.cuisineSpecialty.map(
+                                          (specialty) => (
+                                              <Grid item key={specialty}>
+                                                  <Box
+                                                      className={classes.box}
+                                                      p={1}
+                                                      fontWeight={"fontWeightBold"}
+                                                  >
+                                                      <Typography>
+                                                          {specialty}
+                                                      </Typography>
+                                                  </Box>
+                                              </Grid>
+                                          )
+                                      )
+                                    : null}
                             </Grid>
                             <Grid item>
-                                <Box mt={3} mb={3}>
+                                <Box mt={1} mb={1}>
                                     <Divider className={classes.line} />
                                 </Box>
                             </Grid>
                             <Grid item>
-                                <Box mb={2}>
+                                <Box mb={3}>
                                     <Typography variant="body1">
-                                        {user.profile.bio}
+                                        {chefInfo.bio}
                                     </Typography>
                                 </Box>
                             </Grid>
                         </Grid>
                     </div>
                     <div>
-                        <Button
-                            color="primary"
-                            variant="contained"
-                            className={classes.footer}
-                        >
-                            Send Request
-                        </Button>
+                        {chefInfo.chefProfile._id !== currentChef._id ? (
+                            <Button
```

Comment:
What does/will this button do?

Reply:
Right now nothing.  It was in the original docs so I kept it in incase we want to rework this page so that it will be more like the ChefProfile page... meaning if they go to this page and they aren't the user you won't be able to edit and you can can send the person a message.